protoc-gen-gqlgencfg/main.go

```diff
@@ -70,21 +85,38 @@ func (p *plugin) Generate(file *generator.FileDescriptor) {
 
 			cfg := &config{
 				SchemaFilename: []string{},
-				Exec:           packageConfig{Filename: "exec.go"},
-				Model:          packageConfig{Filename: "models.go"},
+				Exec:           packageConfig{Filename: path.Join(p.Gendir, "generated.gen.go")},
```

Comment:
I thing name `exec` better suits this file than 'gnerated'. Also `.gen` already stands for generated